plugin/pkg/scheduler/api/v1/types.go

```diff
@@ -32,6 +32,10 @@ type Policy struct {
 	Priorities []PriorityPolicy `json:"priorities"`
 	// Holds the information to communicate with the extender(s)
 	ExtenderConfigs []ExtenderConfig `json:"extenders"`
+	// RequiredDuringScheduling affinity is not symmetric, but there is an implicit PreferredDuringScheduling affinity rule
+	// corresponding to every RequiredDuringScheduling affinity rule.
+	// HardPodAffinitySymmetricWeight represents the weight of implicit PreferredDuringScheduling affinity rule, in the range 0-100.
+	HardPodAffinitySymmetricWeight int
```

Comment:
It's a kind of moving its location: a new parameters in policy, it can override the value from cli. prefer to `*int`, as we can distinguish "set to 0" or "not set".